CMakeLists.txt

```diff
@@ -79,8 +79,8 @@ set(CMAKE_CXX_FLAGS_UBSAN
 	CACHE STRING "Flags used by the C++ compiler during UndefinedBehaviourSanitizer builds."
 	FORCE)
 
-add_subdirectory(lib/)
-add_subdirectory(use_case/)
+add_subdirectory(lib)
```

Comment:
I don't think this is necessary change. When wring path/ it is easier (for me to spot dirs)